src/models/event.ts

```diff
@@ -394,5 +413,17 @@ export default class Event extends RestfulModel {
         'The number of participants in the event exceeds the set capacity.'
       );
     }
+
+    if (
+      this.id &&
+      ((this.reminders &&
+        (this.reminders.reminderMethod || this.reminders.reminderMinutes)) ||
```

Comment:
Why do we throw an error if the event has a reminders object set? In this case the user can never update an event that was created with a reminder as ID will be set and `this.reminders` will be set.